docs/start/overview.md

```diff
@@ -28,13 +28,13 @@ If you'd like to help with adding Windows support, please [say hello in this iss
 
 Building a Jupyter Book broadly consists of two steps:
 
-1. **Put your book content in a folder**. Jupyter Book needs the following
+1. **Put your book content in a folder or a file**. Jupyter Book needs the following
```

Comment:
```suggestion
1. **Put your book content in a folder**. Jupyter Book needs the following
```

I think that for the "getting started" section we should still just focus on a book use-case, and underneath we can add a `{note}` block that mentions you can build individual files as well. what do you think?